src/index.js

```diff
@@ -2,17 +2,20 @@ import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/App';
 import { Provider } from 'react-redux';
-import { createStore, compose } from 'redux';
+import { createStore, compose, applyMiddleware } from 'redux';
+import { createEpicMiddleware } from 'redux-observable'
 import rootReducer from './reducers';
+import rootEpic from './epics';
 import ReduxDevTools from './components/_dev/ReduxDevTools';
 import 'isomorphic-fetch';
```

Comment:
@Spetastian What does this do?

Reply:
It imports whatever epics/index.js exports which is all the epics combined into one root epic. We only have one epic atm but this will be the file were we add new ones after we have created them in the epics folder.

Reply:
It imports whatever epics/index.js exports which is all the epics combined into one root epic. We only have one epic atm but this will be the file were we add new ones after we have created them in the epics folder.

Reply:
This is not suppose to be there anymore. But basically `fetch` is just a browser feature not a JavaScript feature. So if you want to write tests for that or run it on the server you will have to use one of these `isomorphic` modules that uses fetch in the browser but polyfills it if it does not exist.

Since we are using RxJS Ajax feature we do not need this at all.